integration_tests/src/main/python/parquet_test.py

```diff
@@ -685,3 +688,351 @@ def test_parquet_reading_from_unaligned_pages_basic_filters_with_nulls(spark_tmp
         assert_gpu_and_cpu_are_equal_collect(
                 lambda spark : spark.read.parquet(data_path).filter(filter_str),
                 all_confs)
+
+
+
+@pytest.mark.skipif(is_before_spark_330(), reason='this is a new feature of Spark 330')
+@allow_non_gpu(any = True)
+def test_parquet_max_top_column_not_push_down(spark_tmp_path):
+    data_path = spark_tmp_path + "/pushdown.parquet"
+
+    def do_explain(spark):
+
+        data = map(lambda i: ((i, ["val_{}".format(i)]),), range(1, 11))
+        spark.createDataFrame(data).write.mode('overwrite').parquet(data_path)
+        
+        df = spark.read.parquet(data_path).selectExpr("Max(_1)")
+        explain = df._sc._jvm.PythonSQLUtils.explainString(df._jdf.queryExecution(), "simple")
+
+        assert "PushedAggregation: []" in explain
+        return df
+
+    assert_gpu_and_cpu_are_equal_collect(do_explain, {"spark.sql.parquet.aggregatePushdown": "true", 
+                                  "spark.sql.sources.useV1SourceList": ""})
```

Comment:
Nit: It would be nice to factor out the configs specific to aggregate pushdown into a variable that can be reused in the various aggregate pushdown tests to simplify those tests, simplify maintenance, and reduce the chance of config typos.

Reply:
Thank you for the advice! I'll refactor them.

Reply:
done!